NuKeeper/Commands/RepositoryCommand.cs

```diff
@@ -7,12 +7,12 @@
 
 namespace NuKeeper.Commands
 {
-    [Command(Description = "Performs version checks and generates pull requests for a single repository.")]
+    [Command(Description = "Performs version checks and generates pull requests for a single repository on GitHub.")]
     internal class RepositoryCommand : GitHubNuKeeperCommand
     {
         private readonly GithubEngine _engine;
 
-        [Argument(0, Name = "GitHub repository uri", Description = "The repository to scan.")]
+        [Argument(0, Name = "GitHub repository URI", Description = "The URI of the repository to scan.")]
         // ReSharper disable once UnassignedGetOnlyAutoProperty
         // ReSharper disable once MemberCanBePrivate.Global
         protected Uri GitHubRepositoryUri { get; }
```

Comment:
That one's a bit inconsistent now, as `GithubApiEndpoint` got changed from `Uri` to `string` to use `TryParse`

Reply:
Separate I guess. I kind of like having CommnadLineUtils handling the whole "parse into URI" logic, but I guess having more friendly error message could be worth doing this more "manually".